src/matcher.ts

```diff
@@ -139,7 +183,8 @@ export class Matcher {
   /**
    * Match one or more variable bindings.
    */
-  matchBindings(): any { // TODO:
+  matchBindings(): any {
+    // TODO:
```

Comment:
Is there supposed to be a todo here?

Reply:
Looks like an old comment that got repositioned when the file was reformatted. I'll clean it up when I do the final merge!